QKSMS/src/main/java/com/moez/QKSMS/ui/conversationlist/ConversationListFragment.java

```diff
@@ -195,6 +196,9 @@ public void onItemLongClick(final Conversation conversation, View view) {
 
         dialog.addMenuItem(R.string.menu_delete_conversation, MENU_DELETE_CONVERSATION);
 
+        if(conversation.hasError())
+            dialog.addMenuItem(R.string.delete_failed, MENU_DELETE_FAILED);
```

Comment:
Part of the code style for the project is to always use braces for if statements, and there should be a space after `if`. Just like how I posted

Sorry, initially this is gonna be kind of annoying but I'm just trying to keep things consistent haha